app/src/main/java/ch/epfl/sdp/blindly/main_screen/MainScreen.kt

```diff
@@ -45,4 +47,21 @@ class MainScreen : AppCompatActivity() {
             }
         })
     }
+
+    override fun onBackPressed() {
+        val builder: AlertDialog.Builder = AlertDialog.Builder(this)
+        builder.setTitle("Exit")
+        builder.setMessage("Are You Sure?")
```

Comment:
Does the message say anything else than "Are you Sure?" ? I think it could be better to inform the user that doing so will close the app. Otherwise it might be taken as a "Are you sure you want to go back to the sign up part?"

Reply:
Check the line 53 :D But if you want, we can further specify further as "Exit the app." instead of Exit.

Reply:
Haha yeah, mb! But I think it would be clearer to add the app :D